### PR TITLE
Skip node in scale-down if it was recently found unremovable

### PR DESCRIPTION
**Pull request by MaciekPytel**
Ref: #245

### CODE REVIEW

File: cluster-autoscaler/core/scale_down.go [hunk 1]
```diff
@@ -124,9 +128,25 @@ func (sd *ScaleDown) UpdateUnneededNodes(
...
+			if unremovableTimestamp.After(timestamp) {
+				continue
+			}
+			delete(sd.unremovableNodes, node.Name)
+		}
+		nodesToCheck = append(nodesToCheck, node)
+	}
+	skipped := len(managedNodes) - len(nodesToCheck)
+	if skipped > 0 {
+		glog.V(1).Infof("Ignoring %v recently checked nodes in scale-down calculation", skipped)
```
Comment (mwielgus) [inline cluster-autoscaler/core/scale_down.go]: This comment is misleading. What about something like: 
scale-down calculations: ignoring %v nodes, that were unremovable in the last %s minutes

File: cluster-autoscaler/core/scale_down.go [hunk 2]
```diff
@@ -191,6 +211,15 @@ func (sd *ScaleDown) UpdateUnneededNodes(
 		}
 	}
 
+	// Update unremovable map
```
Comment (mwielgus) [inline cluster-autoscaler/core/scale_down.go]: Change comment to: add new nodes to unremovable map

File: cluster-autoscaler/core/scale_down.go (hunk 1 above)
Comment (MaciekPytel) [reply cluster-autoscaler/core/scale_down.go]: done

File: cluster-autoscaler/core/scale_down.go (hunk 2 above)
Comment (MaciekPytel) [reply cluster-autoscaler/core/scale_down.go]: done

**Pull request merged**